hl/src/H5LTanalyze.l

```diff
@@ -123,7 +123,7 @@ OPQ_TAG          {return token(OPQ_TAG_TOKEN);}
 ":"              {return token(':');}
 ";"              {return token(';');}
 [ \t\n]*         ;
-"\n"             { return 0; }
+"[\n]"             { return 0; }
```

Comment:
BTW, I am pretty sure this new rule matches a newline in literal square brackets.

Without the double quotes, the rule would match a literal newline, and `flex` should warn that the rule will never match, because line 125 matches newlines, also, with higher priority.